tests/Doctrine/Migrations/Tests/Tools/Console/Command/MigrateCommandTest.php

```diff
@@ -340,9 +381,28 @@ protected function setUp() : void
 
         $this->migrateCommandTester = new CommandTester($this->migrateCommand);
 
-        $this->storage = new TableMetadataStorage($connection, $this->configuration->getMetadataStorageConfiguration(), $this->migrationRepository);
+        $this->storage = new TableMetadataStorage($this->connection, $this->metadataConfiguration, $this->migrationRepository);
         $this->storage->ensureInitialized();
 
         $this->dependencyFactory->setService(MetadataStorage::class, $this->storage);
     }
+
+    private function alterMetadataTable() : void
+    {
+        $originalTable = $this->connection->getSchemaManager()
+            ->listTableDetails($this->metadataConfiguration->getTableName());
+
+        $modifiedTable = $this->connection->getSchemaManager()
+            ->listTableDetails($this->metadataConfiguration->getTableName());
```

Comment:
why listing table details twice rather than cloning the Table object ?

Reply:
wasn't sure that `clone` will deep clone the required props, but will try to use that.

Reply:
seems working, fixed, thanks